### ISSUE TITLE
Addon's content script should delve into iframes

### ISSUE THREAD

**Issue by toolness** [mozilla/webmaker-addons#26]
Currently the addon's content script only goes into the top-level page elements, which is super lame.

**Comment by xmatthewx** [comment]
Good catch. For now, that's probably ok. We can improve things like this after the add-on proves useful.

**Comment by toolness** [comment]
What happened to your vacation?!! :tropical_drink:
